modules/virt-importing-vm.adoc

```diff
@@ -0,0 +1,121 @@
+[id='virt-importing-vm_{context}']
+ifdef::virt-importing-vmware-vm[]
+= Importing a VMware virtual machine or template
+
+You can import a VMware virtual machine or template using the virtual machine wizard.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+= Importing a Red Hat Virtualization virtual machine with the virtual machine wizard
+
+You can import a Red Hat Virtualization (RHV) virtual machine using the virtual machine wizard.
+endif::[]
+
+.Procedure
+
+. In the web console, click *Workloads* -> *Virtual Machines*.
+. Click *Create Virtual Machine* and select *Import with Wizard*.
+
+ifdef::virt-importing-vmware-vm[]
+. In the *Connect to Provider* screen, select *VMware* from the *Provider* list.
+. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
+
+* If you select *Connect to New Instance*, enter the *vCenter hostname*, *Username*, and *Password*.
+* If you select a saved vCenter instance, the wizard connects to the vCenter instance using the saved credentials.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+. In the *Connect to Provider* screen, select *Red Hat Virtualization (RHV)* from the *Provider* list.
+. Select *Connect to New Instance* or a saved RHV instance from the *RHV Instance* list.
+
+* If you select *Connect to New Instance*, fill in the following fields:
+
+** *API URL*: For example, `\https://<RHV_Manager_FQDN>:8443/ovirt-engine/api`
+** *CA certificate*: Click *Browse* to upload the RHV Manager CA certificate or paste the CA certificate into the field.
++
+You can obtain the RHV Manager CA certificate with the following command:
++
+----
+$ curl \
+--output ca.crt \
+'http://<RHV_Manager_FQDN>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA'
+----
+
+** *Username*: RHV Manager user name in the following format: `<username>@<domain>`
+** *Password*: RHV Manager password
+
+* If you select a saved RHV instance, the wizard connects to the RHV instance using the saved credentials.
+endif::[]
+
+. Click *Check and Save* and wait for the connection to complete.
+
+ifdef::virt-importing-rhv-vm[]
+. Select a cluster and a virtual machine to import.
+endif::[]
+ifdef::virt-importing-vmware-vm[]
+. Select a virtual machine or a template to import.
+endif::[]
+
+. Click *Next*.
+. In the *Review* screen, review your settings.
+
+ifdef::virt-importing-rhv-vm[]
+. Optional: You can select *Start virtual machine on creation*.
+endif::[]
+
+. Optional: Click *Edit* to update the following settings:
+
+* *General*:
+
+ifdef::virt-importing-rhv-vm[]
+** *Name*: You must update the virtual machine name if the same name is being used by another virtual machine in the same namespace.
+** *Description*: Optional
+
+* *Storage* > *Storage Class*: If you do not select a storage class, {VirtProductName} uses the default storage class. If a default storage class does not exist, the virtual disk's PVC is bound to a PV without a specified storage class.
```

Comment:
I am not sure no default is a possible option in CNV. Also if indeed it's possible - I think it will not bind to a PV, as the storage class is not defined.
Would you please check that with devs?

Reply:
@pkliczewski @suomiy 
One of you mentioned the behavior of importing VM with no default storage class defined. Can you confirm?

Reply:
I found this discussion:

<img width="377" alt="image" src="https://user-images.githubusercontent.com/31725355/87915129-6191fe00-ca7a-11ea-8e28-75c92cd67405.png">

Reply:
@suomiy  Confirmed that the current text is correct. Imported VM's PV will have no specified storage class.